dds/DCPS/Qos_Helper.cpp

```diff
@@ -6,9 +6,477 @@
  */
 
 #include "DCPS/DdsDcps_pch.h" //Only the _pch include should start with DCPS/
-#include "Qos_Helper.h"
-#include "Service_Participant.h"
 
+#include "Qos_Helper.h"
 #if !defined (__ACE_INLINE__)
 #include "Qos_Helper.inl"
 #endif /* __ACE_INLINE__ */
+
+#include "debug.h"
+#include "Service_Participant.h"
+
+bool Qos_Helper::valid(const DDS::DomainParticipantQos& qos)
```

Comment:
Will need to put the declarations in the `OpenDDS::DCPS` namespace. This will need to include the version namespace macro. Here is an example:

https://github.com/objectcomputing/OpenDDS/blob/eba3661e705994915f7ca8486d2182509029d9b4/dds/DCPS/DCPS_Utils.cpp#L24-L27

https://github.com/objectcomputing/OpenDDS/blob/eba3661e705994915f7ca8486d2182509029d9b4/dds/DCPS/DCPS_Utils.cpp#L527-L530